docs/_ext/spelling_stub_ext.py

```diff
@@ -22,9 +22,9 @@ def run(self) -> List[nodes.Node]:
 
 def setup(app: Sphinx) -> Dict[str, Union[bool, str]]:
     """Initialize the extension."""
-    app.add_directive('spelling', SpellingNoOpDirective)
+    app.add_directive("spelling", SpellingNoOpDirective)
```

Comment:
It was probably a bad idea to format a file that is copied over from other projects.